app/controllers/admin/reports_controller.rb

```diff
@@ -61,6 +61,9 @@ def render_data?
     def render_in_background
       cable_ready[ScopedChannel.for_id(params[:uuid])]
         .inner_html(
+          selector: "#report-go",
+          html: helpers.button(t(:go), "report__submit-btn", "submit", disabled: true)
```

Comment:
I don't think that this is actually needed because we do it client-side. But it doesn't hurt either.